src/activities/settings/SettingsActivity.cpp

```diff
@@ -20,6 +20,7 @@ const SettingInfo settingsList[settingsCount] = {
      SettingType::ENUM,
      &CrossPointSettings::orientation,
      {"Portrait", "Landscape CW", "Inverted", "Landscape CCW"}},
+    {"Swap Front Buttons", SettingType::TOGGLE, &CrossPointSettings::swapFrontButtons, {}},
```

Comment:
"Swap" is a but unclear, even if were not doing the side buttons, there are a few different configurations which we may not want to add in this PR, but starting as an enum (even just the two values) will make this easier to expand.

Something like, "Front Button Layout", with the two string values shown to users as "Bck, Cnfrm, Lft, Rght", and "Lft, Rght, Bck, Cnfrm" so that it's more clear what the setting is actually affecting